libraries/wallet/wallet.cpp

```diff
@@ -1860,11 +1900,7 @@ fc::ecc::private_key wallet_api::derive_private_key(const std::string& prefix_st
         }
 
         annotated_signed_transaction wallet_api::delegate_vesting_shares(string delegator, string delegatee, asset vesting_shares, bool broadcast) {
-            FC_ASSERT(!is_locked());
-            auto accounts = my->_remote_database_api->get_accounts({delegator, delegatee});
-            FC_ASSERT(accounts.size() == 2, "One or more of the accounts specified do not exist.");
-            FC_ASSERT(delegator == accounts[0].name, "Delegator account is not right?");
-            FC_ASSERT(delegatee == accounts[1].name, "Delegatee account is not right?");
```

Comment:
Why did you remove this checks?

Reply:
In the other places this check is necessary, because the account data is used for the correct creation of the request.

Reply:
So as I understood right, all other checks don't removed (`account = get_account()`) only because `account` variable used later in code. Ok.

Reply:
> In the other places this check is necessary, because the account data is used for the correct creation of the request.

just wrote the same : )